Classes/DataProcessing/ContainerProcessor.php

```diff
@@ -83,15 +83,11 @@ public function process(
             } else {
                 $colPos = (int)$processorConfiguration['colPos'];
             }
-            $as = 'children';
-            if ($processorConfiguration['as']) {
-                $as = $processorConfiguration['as'];
-            }
```

Comment:
i would prefer to keep the `$as` as a variable (for better readability): `$as = $cObj->stdWrapValue('as', $processorConfiguration, 'children');`

Reply:
@achimfritz Please have a look at it again